core/src/smartcontracts/wasm.rs

```diff
@@ -119,9 +122,15 @@ pub mod error {
 
     /// `WebAssembly` execution error type
     #[derive(Debug, thiserror::Error, displaydoc::Display)]
+    #[ignore_extra_doc_attributes]
```

Comment:
why?

Reply:
It's for `displaydoc::Display` derive. Need because I don't want to pollute user-friendly message with developer-specific info described in the next paragraph for `Finalization` error